static/app/components/draggableTabs/draggableTab.tsx

```diff
@@ -73,28 +112,80 @@ function BaseTab(
     [to, orientation]
   );
 
+  const {dropProps, isDropTarget} = useDroppableItem(
+    {
+      target: {type: 'item', key: item.key, dropPosition: 'on'},
+    },
+    dropState,
+    ref
+  );
+
+  function Draggable({children}) {
+    const {dragProps, dragButtonProps, isDragging} = useDrag({
+      getAllowedDropOperations: () => ['move'],
```

Comment:
you'll need to pull Draggable out of this component since things like calling hooks inside here will violate rules of hooks